.github/workflows/test.yml

```
@@ -41,4 +41,6 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
       - uses: psf/black@21.12b0
```

Should we use the stable tag here? 

```suggestion
      - uses: psf/black@stable
```